### ISSUE TITLE
Add one KDD 2023 paper

### ISSUE THREAD

**Issue by EdisonLeeeee** [YuanchenBei/Awesome-Pretraining-for-Graph-Neural-Networks#1]
Hi! Thanks a lot for curating this very helpful collection of pertaining graph neural networks papers.

I wanted to ask if you could add the following paper from our group:
+ Title: What's Behind the Mask: Understanding Masked Graph Modeling for Graph Autoencoders
+ Link: https://arxiv.org/abs/2205.10053
+ Code: https://github.com/EdisonLeeeee/MaskGAE

Thank you!

**Comment by YuanchenBei** [comment]
> Hi! Thanks a lot for curating this very helpful collection of pertaining graph neural networks papers.
> 
> I wanted to ask if you could add the following paper from our group:
> 
> * Title: What's Behind the Mask: Understanding Masked Graph Modeling for Graph Autoencoders
> * Link: https://arxiv.org/abs/2205.10053
> * Code: https://github.com/EdisonLeeeee/MaskGAE
> 
> Thank you!

Hi, thanks for providing this insightful paper related to GNN pre-training!  

I have added it to the paper list. Welcome to provide any additional relevant papers!

**Comment by EdisonLeeeee** [comment]
Thank you!
